src/client/store/data/reducers/toggleCompareHistoryItem.ts

```diff
@@ -0,0 +1,23 @@
+import { Draft, PayloadAction } from '@reduxjs/toolkit'
+import { Objects } from 'utils/objects'
+
+import { ActivityLog } from 'meta/assessment'
+import { HistoryTarget } from 'meta/cycleData'
+
+import { DataState } from 'client/store/data/state'
+
+type Action = PayloadAction<{
+  datum: ActivityLog<never>
+  target: HistoryTarget
+}>
+
+export const toggleCompareHistoryItem = (state: Draft<DataState>, action: Action) => {
+  const { datum, target } = action.payload
+
+  if (state.history.compareItem?.[target]?.id === datum.id) {
```

Comment:
better uuid ?

Reply:
If we have, check please
If not, should we migrate ?

either way, next pr is fine